sources/components/markdown/MermaidRenderer.tsx

```
@@ -0,0 +1,218 @@
+                <View style={[style.container, style.loadingContainer]}>
+                    <View style={style.loadingPlaceholder} />
+                </View>
+            );
+        }
+
+        return (
+            <View style={style.container}>
+                {/* @ts-ignore - Web only */}
+                <div
+                    style={webStyle}
+                    dangerouslySetInnerHTML={{ __html: svgContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)